### DISCUSSION TITLE
Totally lost in how to load HTML dynamically

### DISCUSSION THREAD

**Discussion by Thallius** [parcel-bundler/parcel#6805] [Q&A]
Hi,

I have a single page application which is loading HTML templates dynamically into the DOM.

At the moment I use $.get() from jquery but everything else would be fine too. e.G.

`

    load(destination, file, replace, display, func)
    {
        console.log("SSP Load "+file);

        $.get( file + ".html?"+this.version, (data) => this.loaded(data, destination, file, replace, display, func))
        .fail(() => alert("Sorry unable to load resource "+file))
        .always()
        .then();
    }

    loaded(data, destination, file, replace, display, func)
    {
       // Here I manipulate the HTML string and then 
       destination.append(data);
    }

`

But the content of data is always the content of my index.html file
I think its because Parcel isn't aware of the xy.html.

So how can I load HTML templates into my DOM with Parcel?

Thanks

Claus

**Comment by ghost** [comment]
Hello,

I use this import to get templates :

`import html from 'bundle-text:../assets/templates/user.html'`

My templates have : 

<template></template>

https://v2.parceljs.org/features/bundle-inlining/

Eric
